geode-core/src/main/java/org/apache/geode/distributed/ConfigurationProperties.java

```diff
@@ -1916,11 +1916,10 @@
   /**
    * The static String definition of the <i>"redis-port"</i> property <a name="redis-port"/a>
    * </p>
-   * <U>Description</U>: Specifies the port used by {@code GeodeRedisServer} which enables redis
-   * clients to connect and store data in GemFire distributed system. see {@code GeodeRedisServer}
-   * for other configuration options. A value of "-1" will select a random port.
+   * <U>Description</U>: Specifies the port used by Redis API for Geode which enables redis
+   * clients to connect and store data in GemFire distributed system.
```

Comment:
I think you should specify here what special values do. We had some of this text before which may have been lost in a merge. For example:
A value of "0" disables the Redis API for Geode.
A value of "-1" will select a random available port.

Reply:
good catch, I'll update that.

Reply:
These selections are consistent with network services though. Zero typically mean automatically pick an ephemeral port. Given that ports should not be negative, can we use another property or API to enable service, or the use of `null`.

Reply:
@pivotal-jbarrett That's true. `-1` was chosen because `0` is already used to indicate that we don't want the service to be enabled. However we could (should?) switch that notion now while things are still `Experimental`.

Reply:
Created a Jira for it: https://issues.apache.org/jira/browse/GEODE-8139
Will either do it now or add a story in the backlog